utils/sql-db-seeder/src/mongo-db-client/mongo-db-client.ts

```diff
@@ -0,0 +1,23 @@
+import z from 'zod';
+import { MongoDbClient } from '@ukef/dtfs2-common/mongo-db-client';
+
+const { MONGODB_URI_QA, MONGO_INITDB_DATABASE } = process.env;
+
+const mongoDbConfigSchema = z.object({
+  MONGODB_URI_QA: z.string(),
+  MONGO_INITDB_DATABASE: z.string(),
+});
+
+/**
+ * Gets the mongo db client
+ * @returns The mongo db client
+ */
+const getMongoDbClient = (): MongoDbClient => {
+  const mongoDbConfig = mongoDbConfigSchema.parse({ MONGODB_URI_QA, MONGO_INITDB_DATABASE });
+  return new MongoDbClient({
+    dbName: mongoDbConfig.MONGO_INITDB_DATABASE,
+    dbConnectionString: mongoDbConfig.MONGODB_URI_QA,
+  });
+};
```

Comment:
Why are we duplicating this?
<img width="397" alt="Screenshot 2024-09-09 at 11 44 33" src="https://github.com/user-attachments/assets/8419c61d-76c3-46ac-ac56-ff42d16f4599">

Reply:
I'm not sure why because I didn't write this, but I don't think it matters too much given this is just within the utils folder. I do think different utils scripts should own their own connection parameters so I'm not sure I want to pull it out as is, but also I don't think we should spend more time engineering this further given it is just in utils.

Reply:
Utils or not, concerned that the same DB client is just getting copied and pasted, it's unnecessary and introduces maintenance overhead. E.g, if the client needs updating we have to update it in at least 3 places.

Not a blocker for this PR since this PR is blocking things and has been open for a while - perhaps we can create a tech debt ticket to create a DRY utils mongo DB client?

Reply:
Tech debt ticket created in backlog here: https://ukef-dtfs.atlassian.net/browse/FN-3438